spec/features/thredded/user_deletes_post_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'spec_helper'
+
+feature 'User deleting posts' do
+  scenario 'can delete their own post' do
+    user.log_in
+    topic = users_topic
+    topic.visit_topic
+    last_post = last_post(topic)
+
+    expect(last_post).to be_listed
+    expect(last_post).to be_deletable
+
+    last_post.delete
+
+    expect(last_post).to_not be_listed
+  end
+
+  scenario "cannot delete someone else's post" do
+    user.log_in
+    topic = someone_elses_topic
+    topic.visit_topic
+    last_post = last_post(topic)
+
+    expect(last_post).to be_listed
+    expect(last_post).to_not be_deletable
```

Comment:
... which would then remove the need here to assign it to a variable. `expect(topic.last_post)` could be used for both of these here.